wgpu-types/src/lib.rs

```diff
@@ -1574,7 +1614,7 @@ pub struct RenderPipelineDescriptor<'a, L, D> {
     /// Bitmask that restricts the samples of a pixel modified by this pipeline. All samples
     /// can be enabled using the value `!0`
     pub sample_mask: u32,
-    /// When enabled, produces another sample mask per pixel based on the alpha output value, that
+    /// When enabled, produces another sample mask per pixel based on theT alpha output value, that
```

Comment:
Is this intentional?

Reply:
No, just missed it when quickly committing.